app/controllers/questions/start_controller.rb

```diff
@@ -22,5 +24,11 @@ def submit
     def start_form_params
       params.require(:start_form).permit!
     end
+
+    def store_form_in_session!
```

Comment:
Since this is a question of "continue or exit", do we need to store this value in the session? How will we use it later?

Reply:
See below.